src/Seq.Input.HealthCheck/HealthCheckResult.cs

```diff
@@ -33,6 +34,8 @@ class HealthCheckResult
         public string MessageTemplate { get; } =
             "Health check {Method} {TargetUrl} {Outcome} with status code {StatusCode} in {Elapsed:0.000} ms";
 
+        private const string _messageTemplateForRedirect = "Health check {Method} {TargetUrl} redirected {RedirectCount} time(s) to {FinalUrl} {Outcome} with status code {StatusCode} in {Elapsed:0.000} ms";
```

Comment:
Although I think the customized message template reads nicely, this will change the underlying "event type", so dashboard charts etc. that target health check events won't pick up redirect results without modification/special casing. Might be best to just stick with the original template and add the additional properties to the event payload only.